docs/reference/mapping/types/runtime.asciidoc

```diff
@@ -0,0 +1,94 @@
+[[runtime]]
+=== Runtime
+
+////
+[source,console]
+----
+PUT /my-index
+----
+// TESTSETUP
+////
+
+Typically, you must index fields to {es} before they can be retrieved,
+aggregated, or searched. With runtime fields, you can explicitly define a field
+in the mapping and access it at runtime without indexing your data. This
+flexibility allows you to more quickly ingest raw data into the Elastic Stack
+and immediately access it. By dynamically evaluating runtime fields at query
+time, you can choose which fields to index and optimize disk space.
+
+include::{es-ref-dir}/mapping/types.asciidoc[tag=runtime-fields-description]
+
+// Each runtime field is of the `runtime` data type, and has its own
+// <<mapping-types,field type>>, such as `boolean`, `long`, or `keyword`. The
+// field type identifies the data type In
+// the following example, the data type is `runtime` and the runtime field type is
+// `keyword`.
+
+==== Specifying runtime fields
+Runtime fields do not have access to the
+<<mapping-source-field,`_source` field>>. You specify runtime fields in the
+mapping by <<modules-scripting-using,defining a script>>, which always has
+access to the `source` field. At search time, the script runs and generates
```

Comment:
This is true, but it might not be the right time to talk about it. To be honest we'd prefer folks use the `doc` way of accessing fields if at all possible and only reach for `_source` if it is, like, their only choice.